tests/charts/test_basic_helm_chart.py

```diff
@@ -37,8 +37,7 @@ def _get_values_with_version(self, values, version):
         return values
 
     def _get_object_count(self, version):
-        # TODO remove default from condition after airflow update
-        if version == "2.3.2" or version == "default":
+        if version == "2.3.2":
```

Comment:
could do `semver.VersionInfo.parse(version) < (2, 4)` for better clarity

Reply:
After looking at this again with fresh eyes, I like `if version == "2.3.2"` better than `if version != "default" and semver.VersionInfo.parse(version) < (2, 4)`. Seems more straight forward.